### ISSUE TITLE
NVDA reads random info periodically in windows 10

### ISSUE THREAD

**Issue by RichCaloggero** [nvaccess/nvda#9998]
### Steps to reproduce:

I'm not sure how to fill in this section. It seems random. Sometimes it will happen twice in the space of a minute, sometimes it will not happen for a few minutes.

Sometimes it will happen when NVDA is silent and I am not typing or otherwise touching the keyboard. Sometimes it happens while NVDA is reading something in SayAll mode. In this latter case, NVDA stops reading the text it should be reading, reads some random stuff, and then stops.

I've tried changing every setting I can think of in windows 10, but have not been able to stop this from happening.

### Actual behavior:

The random text read seems to be text from the currently focused app. For instance, on this github page, or if I'm in thunderbird thin a message list then it reads a random message list item, etc.

### Expected behavior:

I've never seen this in windows 7.

Ah, it just happened now as I am typing this. The text read was from a few line above this:

>The random text read seems to be text from the currently focused app. For instance, on this 

Perhaps it is due to scrolling, but again, never seen this is windows 7.
 
#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

2019.1.1

#### Windows version:

windows 10

#### Name and version of other software in use when reproducing the issue:

n/a; happens in whatever app has current focus.

#### Does the issue still occur after restarting your PC?

Yes.

**Comment by josephsl** [comment]
Hi, can you try with all add-ons off? I suspect it’s something I need to look at if a specific add-on of mine is involved. Thanks.

**Comment by feerrenrut** [comment]
@RichCaloggero could you please do the following:

1. Change your [log level to debug](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps)
2. Restart NVDA with addons disabled (via Quit NVDA dialog)
3. Reproduce the problem
4. Attach the log file here

This might give us some clues as to what is going on. Bear in mind that when at debug level, the log file will include details of all activity which can be a privacy concern. Hopefully you will be able to reproduce this issue with a short purposeful attempt.

**Comment by DrSooom** [comment]
I suggest to disable mouse tracking by pressing NVDA+M. Maybe he works on a notebook and touches the touchpad by accident. And if a cordless mouse is connected: Charge it.

**Comment by RichCaloggero** [comment]
Here is my debug log:
[nvda-old.log](https://github.com/nvaccess/nvda/files/3448147/nvda-old.log)

Disabling add-ons definately lessened the frequency of occurrence.
 It took a long time for it to happen again, but I think it did happen when I finally began reading the document in firefox at darklang.com.


I hope this helps.

**Comment by RichCaloggero** [comment]
## Here's another significantly shorter log!

What I did:

1. visited the following url in firefox: https://slate.com/technology/2019/07/years-and-years-finale-bethany-transhumanist.html?via=section_recirc_recent
2. moved to the second `h1` tag
3. pressed downArrow to get to list of other articles in category
4. pressed comma to skip the list
5. pressed nvda+downArrow to begin say-all

After reading for about a minute or less, speech stopped with the phrase "Lydia West as Bethany in Years and Years.'".  The phrase seems to be an alt for an image, and was not in the text supposedly being read. The speech also stopped after this phrase was uttered, even though there was more to be read.


This process was also repeatable; I repeated it about 5 times before enabling the debug log.

**Comment by RichCaloggero** [comment]
## Final example

[nvda-1.log](https://github.com/nvaccess/nvda/files/3448849/nvda-1.log)

What I did:

1. visited same url as above: https://slate.com/technology/2019/07/years-and-years-finale-bethany-transhumanist.html?via=section_recirc_recent
2. pressed insert+n to bring up NVDA menu
3. simply waited for about 30 seconds

After about 30 seconds without any input from me, NVDA again spoke the phrase  "Lydia West as Bethany in Years and Years.'".

In the log, search for the string "NVDA  menu".

**Comment by josephsl** [comment]
Hi, have you tried with other browsers? That can help us figure out if this is NVDA, the web browser, or a combination of them. Thanks.

**Comment by DrSooom** [comment]
@RichCaloggero: Please try to reproduce this with eSpeak NG. Thanks.

**Comment by Neurrone** [comment]
Some laptops, unfortunately including mine, have a bug where the mouse would move about the screen randomly even without touch input. This would be the most likely cause.

**Comment by DrSooom** [comment]
In that case, do the following first to figure out if it's really the mouse cursor:
+ Mouse Settings:
  + Enable/Disable "Mouse tracking"
  + Enable/Disable "Play audio coordinates when mouse moves"
+ Review Cursor Settings
  + Enable/Disable "Follow mouse cursor"

And if all these settings don't help, disable the internal mouse/Touchpad/Touch-Point via the Device Manager (Win+R » "devmgmt.msc" » OK; Admin privileges are required). But be careful here and plug in an external wired mouse first – and don't disable that wired mouse too.

**Comment by RichCaloggero** [comment]
## I think it is mouse tracking

- I disabled all add-ons and changed to eSpeakNG with same results
- left add-ons enabled and disabled mouse tracking and seems to fix the issue

My system is a windows 10 desktop machine with a wired mouse. Can anyone tell me why the mouse cursor should randomly move about, even though I was no where near the physical mouse at all?

Thanx for all your suggeshelp and suggestions with this!

-- Rich

**Comment by feerrenrut** [comment]
> Can anyone tell me why the mouse cursor should randomly move about

There are a few potential explanations:
- Vibrations on the desk / table / work surface might cause the mouse to move slightly.
- Sometimes a hair or bit of dust can get in the way of the optical sensor and interfere.

**Comment by XLTechie** [comment]
Is it possible that adjusting the mouse speed/sensitivity down would solve this 
problem?

I have never run into it before on desktops or laptops, but often I have been 
known to turn down pointer speed.

Something else you might try is to return the software to the way it was before 
when the problem was common, and then unplug the mouse. If the problem is gone, 
you know it was this for sure, and not software.

Maybe you just need a new mouse, or a different type.

**Comment by DrSooom** [comment]
@XLTechie: Please enable "Mouse tracking" and "Play audio coordinates when mouse moves" in the Mouse Settings because the beeps are logged in debug mode. If this is the case, then we definitely know that mouse movements cause this issue. Thanks.

**Comment by Adriani90** [comment]
This is definitely not caused by NVDA. I did not hear from other users reporting this problem and I cannot reproduce it myself. I also tend to think that touchpad or mouse is very sensitive.
I am closing this as works for me.
@RichCaloggero if you need further suggestions please contact the users list at
https://nvda.groups.io/g/nvda
Thanks.
